Sources/SKTestSupport/SkipUnless.swift

```diff
@@ -233,6 +233,37 @@ public actor SkipUnless {
     }
   }
 
+  /// Checks if the toolchain contains https://github.com/apple/swift/pull/74080.
```

Comment:
I'm not really sure what to do about this, but the skip test is really the test itself 😅

Reply:
Yes, I know. The key point here is that it won’t skip the test if the toolchain is ≥6.1 or if we are on Swift CI (`SWIFTCI_USE_LOCAL_DEPS ` is set – https://github.com/apple/sourcekit-lsp/blob/main/Documentation/Environment%20Variables.md#build-time)

Reply:
Well, my point was more that it would skip if the original test would fail, though depends on the failure (ie. if *more* results are returned then it wouldn't skip, but if the result is missing at all then it will). Anyway, I think it's fine 👍